STANDALONE_R.md

```diff
@@ -0,0 +1,27 @@
+
+In the case of BST 262, the command to create an .Rprofile file is
+
+```bash
+echo '.libPaths(c("/shared/courseSharedFolders/144013outer/144013/R/x86_64-pc-linux-gnu-library/4.4", .libPaths()))' > ~/.Rprofile
+```
+
+## R Package Installs
+
+R package installs can work in a user home directory, or in a shared directory when the `libPaths()` are set up properly. However, R package installs can also have external dependencies. These external dependencies will come up when installing a package, like the need for `gdal` when installing the R package `raster`. In that case, when R is installed with Spack, the additional depencencies should also be installed with Spack to the same environment where R is installed, providing access to the software.
+
+However, sometimes this isn't enough. In the case of installing `raster`, the dependent package `terra` failed to install because it wasn't able to locate the library files for either `libproj` or `sqlite3`, both of which were installed as Spack packages to the environment via the spack packages `proj` and `sqlite` (see the bst262 spack environment file). I found a [GitHub issue with other people describing the same problem](https://github.com/r-spatial/sf/issues/1471#issuecomment-2148767638), and adapted the install line to `install.packages('terra', configure.args=c("--with-sqlite3-lib=/shared/spack/opt/spack/linux-amzn2-skylake_avx512/gcc-14.1.0/sqlite-3.46.0-qxr7hpygeupogc6rx745sfsfa2ykhws4/lib"))`. It seems that when using `install.packages()` in R, the `configure.args` argument can be used to pass flags to the installer. I got the flag from the GitHub issue, so I'm not sure if there's a way to determine what flags are available in other installers. Another option might be modifying the `LD_LIBRARY_PATHS` variable outside of R, if a similar issue arises and a similar flag for the installer can't be found. In any case, installing the problem dependency with the additional flag worked, which allowed the `raster` package to be installed. Note that passing the additional flag directly to the `raster` install didn't work.
+
+As for the path to the `lib` directory for the `sqlite` dependency installed via spack, that was found with a `spack find -p sqlite` command, which prints out the install path for spack packages. Running `spack find -p` gets all of the paths, and adding a package name limits the output to that package.
```

Comment:
Neat, I was wondering how you determined that library path ([spack find -p](https://spack.readthedocs.io/en/latest/command_index.html#spack-find) is the answer!)